### PR TITLE
First big batch of instructions skeletons

### PR DESCRIPTION
**Pull request by AFOliveira**
This PR has the instructions skeletons of the following extension:
- Zicfiss
- Zifencei
- H
- Zabha
- Zacas
- Zalasr
- Zawrs
- Zfbfmin
- Zicond

Q and D are also almost ready but I am modifying registers naming.

### CODE REVIEW

**Comment by AFOliveira** [comment]
Regression test passed locally, I might have added some folder by mistake, I'll try to fix it asap.

**Review by dhower-qc — CHANGES_REQUESTED** [review]
In general, it seems that every skeleton sets data_independent_timing to true, correct? Could you change that to false? I think that will make it easier to get the correct list in later.

File: arch/inst/H/hinval.gvma.yaml
[the remark this answers is not in the document]
Comment (AFOliveira) [reply arch/inst/H/hinval.gvma.yaml:1]: > In general, it seems that every skeleton sets data_independent_timing to true, correct? Could you change that to false? I think that will make it easier to get the correct list in later.

Of course, will do so now.

>hinval.gvma.yaml is already in the db (under Svinal)

I added on mistake, I think that's what broke the regression test

File: arch/inst/Zifencei/fence.i.yaml
[the remark this answers is not in the document]
Comment (AFOliveira) [reply arch/inst/Zifencei/fence.i.yaml:1]: @dhower-qc  I'm not sure I fully understood this, is it for the fence only or for all instructions?

[the remark this answers is not in the document]
Comment (AFOliveira) [reply arch/inst/Zifencei/fence.i.yaml:1]: Oh, sorry, I didn't notice that. I was trying not to mess with any of the already existing instructions, I'll fix that right now.

**Comment by AFOliveira** [comment]
@dhower-qc  This is related to fence.i not having the parameter "variables", there was none on yours, but I guess now it's breaking the validation. Shall I add an empty "variables" field?
```
Parsing IDL code for RV64...done
Type checking IDL code for _64...

Instructions: |================================================================|

CSRs: |========================================================================|

Functions: |===================================================================|
/home/runner/work/riscv-unified-db/riscv-unified-db/lib/validate.rb:239:in `block in validate_instruction_encoding': undefined method `count' for nil:NilClass (NoMethodError)

        vars_match = variables.count { |variable| ary_from_location(variable["location"]).include?(i) }
                              ^^^^^^
	from /home/runner/work/riscv-unified-db/riscv-unified-db/lib/validate.rb:236:in `times'

```

**Comment by AFOliveira** [comment]
Everything should now be ready.

**Review by dhower-qc — APPROVED** [review]
Thanks!!

**Pull request merged**